app/src/components/pages/collectionLanePage/collectionLanePage.tsx

```diff
@@ -49,19 +72,43 @@ export default function CollectionLanePage() {
           gap: "m",
         }}
       >
-        <Heading sx={{ marginBottom: 0 }} level="h1" text={title} />
+        <Heading sx={{ marginBottom: 0 }} level="h1" id={slug} text={title} />
       </Box>
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      {isLoaded ? (
-        <>
-          <CardsGrid records={mockCollections} />
-        </>
-      ) : (
-        <>
-          <LaneLoading withTitle={false} />,
-          <LaneLoading withTitle={false} />,
-          <LaneLoading withTitle={false} />,
-        </>
+      <Heading
+        size="heading6"
+        sx={{
+          height: "1px",
+          left: "-10000px",
+          overflow: "hidden",
+          width: "1px",
+          _focus: {
+            fontWeight: "400 !important",
+            height: "auto",
```

Comment:
The page will not jump if you can give it a defined height all the time. Otherwise, once you focus away from the heading, the grid jumps up a bit.